src/Microsoft.Extensions.Localization/Internal/AssemblyResourceStringProvider.cs

```diff
@@ -65,6 +63,7 @@ private IList<string> ThrowOrNull(CultureInfo culture, bool throwOnMissing)
         public IList<string> GetAllResourceStrings(CultureInfo culture, bool throwOnMissing)
         {
             var cacheKey = GetResourceCacheKey(culture);
+
```

Comment:
Actually I added a blank line after every `var` statement as usual, can you please let me know when should I add a blank line after `var` coz I saw this in all the repos, let me know if there's a preferred guideline in this case